src/X.PagedList/PagedListExtensions.cs

```diff
@@ -119,31 +119,32 @@ public static IPagedList<T> ToPagedList<T>(this IQueryable<T> superset, int page
         {
             throw new ArgumentOutOfRangeException($"pageSize = {pageSize}. PageSize cannot be less than 1.");
         }
+
+        if (superset == null)
+        {
+            return StaticPagedList<T>.Empty(pageNumber, pageSize);
+        }
+
+        var supersetCount = superset.Count();
         
-        if (totalSetCount.HasValue && superset.Count() > totalSetCount.Value)
+        if (supersetCount > totalSetCount)
```

Comment:
What Happens When `totalSetCount` is null?

Reply:
It should default to `false` all things being equal but just need to be sure that the behaviour is constant

Reply:
We will always have **false** for all comparisons between any number and _null_

Reply:
Awesome.